deployment/dynatrace-azure-forwarder.json

```diff
@@ -95,7 +101,6 @@
             "condition": "[parameters('deployActiveGateContainer')]",
             "name": "[variables('virtualNetworkName')]",
             "location": "[resourceGroup().location]",
-            "tags": "[parameters('resourceTags')]",
```

Comment:
I think conflict resolution went wrong, because all these `tags` lines shouldn't be removed